org-code-javabuilder/playground/src/test/java/org/code/playground/TextItemTest.java

```diff
@@ -77,5 +84,35 @@ public void settersSendChangeMessages() {
     assertEquals(
         Double.toString(newRotation),
         messages.get(4).getDetail().get(ClientMessageDetailKeys.ROTATION));
+    assertEquals(
+        Integer.toString(colorRed),
+        messages.get(5).getDetail().get(ClientMessageDetailKeys.COLOR_RED));
+    assertEquals(
+        Integer.toString(colorGreen),
+        messages.get(6).getDetail().get(ClientMessageDetailKeys.COLOR_GREEN));
+    assertEquals(
+        Integer.toString(colorBlue),
+        messages.get(7).getDetail().get(ClientMessageDetailKeys.COLOR_BLUE));
```

Comment:
This is starting to turn into a pretty big test. Consider splitting the color-related checks into their own test.

Reply:
Agreed - I'll go ahead and split the color checks into their own test.